ui/components/multichain/product-tour-popover/product-tour-popover.scss

```diff
@@ -34,6 +34,16 @@
     top: -7px;
   }
 
+  &__header {
+    position: relative;
+  }
+
+  &__previous-icon {
+    position: absolute;
+    left: 0;
+    top: 0;
+  }
+
```

Comment:
Centers text when previous icon button is present unfortunatley we don't have position utility props for the Box component so need to add them as styles